app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -6,6 +6,15 @@
         orange: "<%= asset_path("logo/logo_orange.png") %>"
       };
     }
+
+    get headshots() {
```

Comment:
hm, I agree that it makes the static pages more complicated, but we only have 4 static pages total, and I'd rather make it more uniform. Would we need something similar for non-static pages if we have a static image though?